pages/common/rsync.md

```diff
@@ -26,12 +26,12 @@
 
 - Recursively copy directories, use archive mode, resolve symlinks and skip files that are newer on the destination:
 
-`rsync --recursive --archive --update --copy-links {{path/to/source}} {{path/to/destination}}`
+`rsync --archive --update --copy-links {{path/to/source}} {{path/to/destination}}`
 
 - Transfer a directory to a remote host running `rsyncd` and delete files on the destination that do not exist on the source:
 
 `rsync --recursive --delete rsync://{{host}}:{{path/to/source}} {{path/to/destination}}`
 
 - Transfer a file over SSH using a different port than the default (22) and show global progress:
 
-`rsync --rsh 'ssh -p {{port}}' --info=progress2 {{host}}:{{path/to/source}} {{path/to/destination}}`
+`rsync --rsh='ssh -p {{port}}' --info=progress2 {{host}}:{{path/to/source}} {{path/to/destination}}`
```

Comment:
I have the same `rsync` version and `--rsh arg` works. Can you try to run it?
```suggestion
`rsync --rsh 'ssh -p {{port}}' --info=progress2 {{host}}:{{path/to/source}} {{path/to/destination}}`
```

Reply:
Same here, tried it on Ubuntu LTS 22.04, Fedora Workstation 38.